local_costmap:
  global_frame: odom
  robot_base_frame: base_link

  obstacle_range: 2.0
  raytrace_range: 2.2
  
  inflation_radius: 1
  cost_scaling_factor: 3.0
  footprint_padding: 0.01

  update_frequency: 10.0
  publish_frequency: 10.0
  transform_tolerance: 0.5  

  static_map: true  
  rolling_window: true
  width: 4.5
  height: 4.5
  resolution: 0.035

